logging/google/cloud/logging/handlers/transports/background_thread.py

```diff
@@ -48,14 +50,22 @@ def _get_many(queue_, max_items=None):
     :param max_items: The maximum number of items to get. If ``None``, then all
         available items in the queue are returned.
 
+    :type max_latency: float
+    :param max_latency: The maximum number of seconds to wait for more than one
+        item from a queue. This number includes the time required to retrieve
+        the first item.
```

Comment:
Suggestion, add this to the end here:

```This means this is effectively the longest amount of time the background thread will hold onto log entries before sending them to the server.```

Reply:
I've added this line, but to the `_Worker` and `BackgroundThreadTransport` docstrings, since the `_get_many` function's current doc string is not tied to logging but kept quite generic.